src/main/java/org/owasp/validator/css/CssHandler.java

```diff
@@ -138,23 +135,22 @@ public CssHandler(Policy policy, LinkedList<URI> embeddedStyleSheets,
 	 * 
 	 * @param policy
 	 *            the policy to use
-	 * @param embeddedStyleSheets
-	 *            the queue of stylesheets imported
 	 * @param errorMessages
```

Comment:
Now that I think about it, can you move this new parameter to the end of the parameter list? That seems more typically when there are extra/optional parameters. That way the two APIs are identical, except 1 extra parameter at the end.